test/src/edu/stanford/nlp/util/AcronymMatcherTest.java

```diff
@@ -27,6 +26,13 @@ public void testFilterStopWords() {
     assertTrue(AcronymMatcher.isAcronym("AAAS", "American Association for the Advancement of Science".split("\\s+")));
   }
 
+  @Test
+  public void testIsAcronymWithBoundaries() {
+    assertFalse(AcronymMatcher.isAcronym("B", "".split("\\s+")));
```

Comment:
Did you know that "".split("\\s+") is a list of length 1 with an empty string in it?  That was super confusing to debug

Reply:
@AngledLuffa. Yes I saw it. I just did not know if we should handle that, if yes where should we handle it?

Reply:
...and it would appear that although it is undocumented, the treatment of the empty string is to be a wildcard

Reply:
Probably by passing in an empty list, since that seems to be the intent of the test

Reply:
Dear @AngledLuffa how would you like to continue? Should I change/drop the test?

Reply:
As mentioned, an empty list would probably work the way you intended for this test.  Up to you whether that's worthwhile